Scene/ChallengeCreateScene/Sources/ChallengeAdditionalInfoInputScene/ChallengeAdditionalInfoInputSceneFactory.swift

```diff
@@ -14,7 +14,19 @@ public protocol ChallengeAdditionalInfoInputScene: AnyObject, Scene {
 }
 
 public struct ChallengeAdditionalInfoInputConfiguration {
-    
+    let challengeName: String
+    let challengeStartDate: String
+    let challengeEndDate: String
```

Comment:
값이 변경되니까 var이지 않을까요 ~?
다른 코드 참고해보니 보통 public var로 선언되어 있네요!
```suggestion
    public var challengeName: String
    public var challengeStartDate: String
    public var challengeEndDate: String
```

Reply:
요거 이전 화면에서 가져온거 주입만 하는거라 상수로 해도 괜찮지 않을까요?
값 변경은 여기에서는 이루어지지 않습니다!

Reply:
아 다른화면과 착각했네요😅 고대로가죠!!